x-pack/plugins/security_solution/public/detection_engine/rule_response_actions/response_actions_list.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useEffect, useRef, useMemo } from 'react';
+import { EuiSpacer } from '@elastic/eui';
+import type { ResponseActionValidatorRef } from './response_actions_form';
+import type { ResponseActionType } from './get_supported_response_actions';
+import { ResponseActionAddButton } from './response_action_add_button';
+import { ResponseActionTypeForm } from './response_action_type_form';
+import type { ArrayItem } from '../../shared_imports';
+import { UseField, useFormContext } from '../../shared_imports';
+
+interface IResponseActionsListProps {
+  items: ArrayItem[];
+  removeItem: (id: number) => void;
+  addItem: () => void;
+  supportedResponseActionTypes: ResponseActionType[];
+  formRef: React.RefObject<ResponseActionValidatorRef>;
+}
+
+const GhostFormField = () => <></>;
+
+// eslint-disable-next-line react/display-name
+export const ResponseActionsList = React.memo(
```

Comment:
to get rid of this eslint disable, you can just declare the const here, set `ResponseActionsList.displayName = 'ResponseActionsList'` below the component definition, and then `export ResponseActionsList` i know it might seem tedious but display name can help debugging sometimes

Reply:
👍